src/scenes/storeScene.js

```diff
@@ -0,0 +1,916 @@
+                    this
+                ),
+
+                new CardObject(
+                    this,
+                    { name: "papier07", url: "sprites/StoreScene/part3/rayon03_03_papier07.png" },
+                    new Phaser.Math.Vector2(-312, 135),
+                    (scene) => scene.takeObject("papier07"),
+                    this
+                ),
+
+                new CardObject(
+                    this,
+                    { name: "papier08", url: "sprites/StoreScene/part3/rayon03_03_papier08.png" },
+                    new Phaser.Math.Vector2(-479, -106),
+                    (scene) => scene.takeObject("papier08"),
+                    this
+                ),
+
+                new CardObject(
+                    this,
+                    { name: "papier09", url: "sprites/StoreScene/part3/rayon03_03_papier09.png" },
+                    new Phaser.Math.Vector2(-302, -106),
+                    (scene) => scene.takeObject("papier09"),
+                    this
+                ),
+
+                new CardObject(
+                    this,
+                    { name: "papier10", url: "sprites/StoreScene/part3/rayon03_03_papier10.png" },
+                    new Phaser.Math.Vector2(-39, -114),
+                    (scene) => scene.takeObject("papier10"),
+                    this
+                ),
+
+                new CardObject(
+                    this,
+                    { name: "papier11", url: "sprites/StoreScene/part3/rayon03_03_papier11.png" },
+                    new Phaser.Math.Vector2(159, -111),
+                    (scene) => scene.takeObject("papier11"),
+                    this
+                ),
+
+                new CardObject(
+                    this,
+                    { name: "papier12", url: "sprites/StoreScene/part3/rayon03_03_papier12.png" },
+                    new Phaser.Math.Vector2(401, -111),
+                    (scene) => scene.takeObject("papier12"),
+                    this
+                ),
+
+          
+            ]
+        );
+        
+        this.checklist_done = {
+          'pate': {
+            position_x: -276,
+            position_y: 234,
+            done: false
+          },
+          'banane': {
+            position_x: -276,
+            position_y: 300,
+            done: false
+          },
+          'pain': {
+            position_x: -276,
+            position_y: 373,
+            done: false
+          },
+          'papier': {
+            position_x: -261,
+            position_y: 422,
+            done: false
+          },
+        }
+        
+        this.cards = [
+            this.firstShelf,
+            this.secondShelf,
+            this.thirdShelf
+        ];
+
+        //Keep track of wich card is displayed
+        this.cardIdx = StoreCards.FIRST_SHELF;
+        this.current_card = this.firstShelf;
+
+        //Create the dialogue controller
+        this.dialogue = new DialogueController(this);
```

Comment:
Do you have dialogue in this scene? If so you should pass the name of the dialogue file (without the extension) as a second parameter.

Reply:
Removed – we’ll see later if we need a monologue